vnext/template/shared-lib/proj/ExperimentalFeatures.props

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <AppProjectExperimentalFeaturesProps Condition="'$(AppProjectExperimentalFeaturesProps)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'windows\ExperimentalFeatures.props'))\windows\ExperimentalFeatures.props</AppProjectExperimentalFeaturesProps>
```

Comment:
Iirc this might have been hooked for changing WinUI version of community modules or something like that?

Reply:
this simplifies things by making sure everyone includes `$(SolutionDir)\ExperimentalFeatures.props` whether that is the app solution, a native module example/testing solution, or they have some setup other than a windows folder.